package.json

```diff
@@ -39,8 +39,7 @@
   "homepage": "https://github.com/eslint/eslint-release#readme",
   "devDependencies": {
     "chai": "^4.2.0",
-    "eslint": "^5.16.0",
-    "eslint-config-eslint": "^5.0.1",
```

Comment:
We were using an old version of `eslint-config-eslint`, the peer deps issue has been fixed in v7.0.0.

Can we upgrade to `7.0.0` instead of removing it? I made PR #54, and it seems to work on Node 15 and Node 16.